src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -0,0 +1,125 @@
+syntax = "proto3";
+
+package com.netflix.compute.validator;
+
+option java_multiple_files = false;
+option java_package = "com.netflix.compute.validator.protogen";
+option java_outer_classname = "ComputeValidator";
+
+option go_package = "validator";
+
+// ----------------------------------------------------------------------------
+// Data model.
+
+// ----------------------------------------------------------------------------
+// GRPC service
+
+service ValidationService {
```

Comment:
So, I think that these are going to be all implemented by different backend services. The networking code will validate SGs + Static IPs. We're planning on having a different service for IAM roles. I don't know where we'll do S3, or image validation yet.

Reply:
Having multiple backends is fine. We still need a facade to provide a single access point API.

Reply:
I don't think that having a single facade should be done, because that's (yet another) intermediary hop. What's the benefit of that, versus the control plane reaching out to N different services?